README.md

```diff
@@ -0,0 +1,198 @@
+
+- User Profiles: Allow users to manage their profiles, update personal information, and view their reservation history.
+  
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```sh
+ - VS Code or any other code editor
+ - Chrome or any other browser
+```
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+```sh
+  cd my-folder
+  git clone https://github.com/anbehindY/furniture-mall-frontend.git
+```
+### Install
+
+Install this project with:
+
+```sh
+
+  cd furniture-mall-frontend
+  npm install
+
+ ```
+ 
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  
+  PORT=3000 npm start
+
+  ```
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Esther Udondian**
+
+- GitHub: [@Click here](https://github.com/eudondian)
+- Twitter: [@Click here](https://twitter.com/EUdondian)
+- LinkedIn: [@Click here](https://www.linkedin.com/in/esther-udondian/)
+
+👤 **Prince Nwuke**
+
+- GitHub: [@prius2055]()
+- Twitter: [@prius2055]()
+- LinkedIn: [@prius2055]()
+
+👤 **Massa zoumanigui**
+
+- GitHub: [@massazoum]()
+- Twitter: [@zoumaniguimass1]()
+- LinkedIn: [LinkedIn]()
+
+👤 **Alfie Yang**
+
+- GitHub: [ @anbehindY]()
+- Twitter: [@yanminthwin]()
+- LinkedIn: [@Alfie Yang]()
+
+👤 **Osman Wako**
+
+- GitHub: [@githubhandle]()
+- Twitter: [@twitterhandle]()
+- LinkedIn: [LinkedIn]()
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- User will be able to secure payment
+- customer support channels such as chat, email, or phone to assist users with inquiries or issues will be added.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/eudondian/hello-rails-front-end/issues).
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+## ⭐️ Show your support <a name="support"></a>
+
+
+If you like this project give me a star
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+We would like to thank Microverse for the learning experience, our  coding partners for all needed coorporation and Murat Korkmaz of Behance for the design.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ]  On the "License" section of your readme, the link is broken because it points to a file that does not exist in your project.

## Tips and extra help

The correct file is `LICENSE`, not `./MIT.md`

Kindly fix the link.

That way, your app will be well documented 💪